qf/enrollment.go

```diff
@@ -136,3 +138,30 @@ func (m *Enrollments) UserIDs() []uint64 {
 	}
 	return userIDs
 }
+
+func (m *Enrollment) CountApprovedSubmissions(submissions []*Submission) {
```

Comment:
Assuming our discussed simplification of the `LastActivityDate` can be done; I think we can rename this method to `UpdateTotalApproved` and we should add a simple doc comment saying something like:

```go
// UpdateTotalApproved updates the total approved assignments for the current enrollment 
// based on the provided submissions.
```

Reply:
Postponed to separate issue.